spec/requests/api/v1/garden_request_spec.rb

```diff
@@ -85,14 +85,16 @@
         headers = {"CONTENT_TYPE" => "application/json"}
         post "/api/v1/gardens", headers: headers, params: JSON.generate(garden_params)
 
-        garden = Garden.last
-        user.gardens << garden
```

Comment:
I think this line was throwing the false positive.

Reply:
Great detective work on this! I think you are right.